[2024-04-08T05:06:47.079+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-04T00:00:00+00:00 [queued]>
[2024-04-08T05:06:47.090+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-04T00:00:00+00:00 [queued]>
[2024-04-08T05:06:47.091+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-04-08T05:06:47.092+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2024-04-08T05:06:47.092+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-04-08T05:06:47.137+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): second_task> on 2021-11-04 00:00:00+00:00
[2024-04-08T05:06:47.142+0000] {standard_task_runner.py:55} INFO - Started process 9170 to run task
[2024-04-08T05:06:47.144+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2021-11-04T00:00:00+00:00', '--job-id', '1189', '--raw', '--subdir', 'DAGS_FOLDER/our_firest_dag.py', '--cfg-path', '/tmp/tmpig3ur6ne']
[2024-04-08T05:06:47.146+0000] {standard_task_runner.py:83} INFO - Job 1189: Subtask second_task
[2024-04-08T05:06:47.215+0000] {task_command.py:388} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2021-11-04T00:00:00+00:00 [running]> on host 3dc1147ef06e
[2024-04-08T05:06:47.297+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Jiseungmin
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-04T00:00:00+00:00
[2024-04-08T05:06:47.299+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-04-08T05:06:47.300+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello i am task2 and this is the task2 task!']
[2024-04-08T05:06:47.309+0000] {subprocess.py:86} INFO - Output:
[2024-04-08T05:06:47.311+0000] {subprocess.py:93} INFO - Hello i am task2 and this is the task2 task!
[2024-04-08T05:06:47.311+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-08T05:06:47.339+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20211104T000000, start_date=20240408T050647, end_date=20240408T050647
[2024-04-08T05:06:47.397+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-04-08T05:06:47.422+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
